package.json

```diff
@@ -92,7 +92,7 @@
     "react-helmet": "^6.1.0",
     "react-test-renderer": "^16.12.0",
     "react-transition-group": "^4.3.0",
-    "sharp": "^0.25.4",
+    "sharp": "^0.29.3",
```

Comment:
same here, arm64 pre-compiled libs started at 0.29.0, and had to install libvips for arm64 on workstation, added comment to README.md about setup.